paperpress.js

```diff
@@ -117,13 +118,40 @@ Paperpress.prototype._fileToItem = function (file) {
 
 	var item = {
 		title: name,
-		slug: slug
+		slug: slug,
+		content: fileContent,
+		path: null
 	}
 
-	item.path = this.pathBuilder(item, file.collectionName)
-	item.content = (fileType === '.md') ? this._marked.render(fileContent) : fileContent
+	if (frontMatter.test(fileContent)) {
+		var contentData = frontMatter(fileContent)
+		var dataAttributes = contentData.attributes
+
+		// Iterate through attributes and merge into the default item
+		for (var k in dataAttributes) {
+			if (dataAttributes.hasOwnProperty(k)) {
```

Comment:
If we check against `item.hasOWnProperty` it will exclude parameters that are not in the `item` object.

This came to my attention while discovering the module, the attributes are not consistent or Paperpress do not has a standard list of allowed attribute keys.

If you review `_directoryToItem`, the function is taking the attributes in a hardcoded way, as you mentioned @rogr  With the current functionality at `_fileToItem`, the user is actually allowed to add extra parameters.

The idea is to ignore everything except `title`, `slug`, `content` and `path`?